webapp/src/shared/components/Button/Button.tsx

```diff
@@ -0,0 +1,43 @@
+import { ReactNode } from 'react';
+import './Button.css';
+
+export enum ButtonTypes {
+  SUBMIT = 'submit',
+  WARNING = 'warning',
+}
+
+type Buttonprops = {
+  buttonType: ButtonTypes;
+  buttonIcon?: JSX.Element;
+  action?: () => void;
+  disabled?: boolean;
+  children: ReactNode;
+};
+
+export default function Button({
+  buttonType,
+  buttonIcon,
+  action,
+  disabled = false,
+  children,
+}: Buttonprops) {
+  let buttonClassName = 'button text-style-3-bold';
+  if (buttonType === ButtonTypes.SUBMIT) {
+    buttonClassName += ' button-submit';
+  } else if (buttonType === ButtonTypes.WARNING) {
+    buttonClassName += ' button-warning';
+  }
```

Comment:
```suggestion
  let className = 'button text-style-3-bold';
  className += ` button-${variant}`;
```
suggestion: Here we could use a template literal.

Reply:
Very nice solution. Done

Reply:
I'll just inline this, see my comment bellow!